### DISCUSSION TITLE
[Projects Beta] There's no way to edit drafts

### DISCUSSION THREAD

**Discussion by wojtekmaj** [community/community#6352] [Projects and Issues] [answered]
There's no way to edit drafts. Once created, in order to edit them, you need to add a new draft and... Retype the text, because you can't select it either.

**Comment by azenMatt** [comment, marked as answer]
Hi @wojtekmaj - that shouldn't be the case ... you should be either able to double click to edit or hit return when you have the cell focused. 

Do you get an error in the console when you try that?

**Reply by wojtekmaj** [reply]
That works for me, thanks - but the discoverability of this is way too low. 

There is a "..." menu which used to have "Edit note" option, in new Projects it's gone. I was under impression that notes are impossible to edit now. At the very least, the "Edit note" option should be back, and either show a message to teach the user the new flow (dblclick) or just trigger the editing mode straight away.

Also, clicking already-selected note doesn't trigger editing. You gotta double click it fast. Click-wait-click pattern should be something users are used to from file select & rename from Windows, where the logic is: is you click filename of a file which is already selected, rename is triggered.

**Reply by azenMatt** [reply]
Thanks for the feedback - it feels like this works as you would expect on the table, but the board isn't quite as user friendly. 

I think adding an edit option into the `...` menu as you mentioned might make a lot of sense here. Let me take that back to the team and see what we can do to make this easier
